modules/arm_plugin/armplg_build.sh

```diff
@@ -190,39 +169,11 @@ if [ "$WITH_OMZ_DEMO" = "ON" ]; then
-mkdir -p $STAGING_DIR/deployment_tools/inference_engine/bin/$ARCHDIR && \
-find $OPENVINO_HOME/bin/$ARCHDIR/$BUILD_TYPE -maxdepth 1 -type f -exec cp -v {} $STAGING_DIR/deployment_tools/inference_engine/bin/$ARCHDIR \; && \
-mkdir -p $STAGING_DIR/deployment_tools/inference_engine/lib/$ARCHDIR && \
-find $OPENVINO_HOME/bin/$ARCHDIR/$BUILD_TYPE/lib -maxdepth 1 -type f -exec cp -v {} $STAGING_DIR/deployment_tools/inference_engine/lib/$ARCHDIR \; && \
-cp -vr $OPENVINO_HOME/bin/$ARCHDIR/$BUILD_TYPE/lib/python_api $STAGING_DIR/python && \
-cp -vr $OPENVINO_HOME/build/share $STAGING_DIR/deployment_tools/inference_engine/ && \
-cp -v $OPENVINO_HOME/inference-engine/scripts/dependencies.* $OPENVINO_HOME/build/dependencies_64.txt $STAGING_DIR/ && \
-mkdir -p $STAGING_DIR/python/${PYTHONVER}/openvino && \
-cp -vr $OPENVINO_HOME/tools $STAGING_DIR/python/${PYTHONVER}/openvino/ && \
-cp -vr $OPENVINO_HOME/inference-engine/include $STAGING_DIR/deployment_tools/inference_engine/ && \
-cp -vr $OPENVINO_HOME/inference-engine/ie_bridges/c/include/c_api $STAGING_DIR/deployment_tools/inference_engine/include/ && \
-mkdir -p $STAGING_DIR/deployment_tools/inference_engine/samples && \
-cp -vr $OPENVINO_HOME/inference-engine/samples $STAGING_DIR/deployment_tools/inference_engine/samples/cpp && \
-cp -vr $OPENVINO_HOME/inference-engine/ie_bridges/c/samples $STAGING_DIR/deployment_tools/inference_engine/samples/c && \
-cp -vr $OPENVINO_HOME/inference-engine/ie_bridges/python/sample $STAGING_DIR/deployment_tools/inference_engine/samples/python && \
-cp -vr $OPENVINO_HOME/inference-engine/tools $STAGING_DIR/deployment_tools/tools && \
-find $OPENVINO_HOME/model-optimizer -maxdepth 1 -type f -exec cp -v {} $STAGING_DIR/deployment_tools/model-optimizer \; && \
-find $OPENVINO_HOME/model-optimizer/.coveragerc -maxdepth 1 -type f -exec cp -v {} $STAGING_DIR/deployment_tools/model-optimizer \; && \
-cp -vr $OPENVINO_HOME/scripts/setupvars $STAGING_DIR/bin && \
-cp -vr $OPENVINO_HOME/scripts/demo $STAGING_DIR/deployment_tools/demo && \
-cp -vr $OPENVINO_HOME/scripts/install_dependencies $STAGING_DIR/install_dependencies && \
-cp -vr $OPENVINO_HOME/inference-engine/tools $STAGING_DIR/deployment_tools/python_tools && \
-(! [ "$WITH_OMZ_DEMO" = "ON" ] || mkdir -p $STAGING_DIR/deployment_tools/inference_engine/demos) && \
-(! [ "$WITH_OMZ_DEMO" = "ON" ] || cp -vr $OMZ_DEMOS_BUILD $STAGING_DIR/deployment_tools/inference_engine/demos) && \
-echo "=================================RPATH cleaning==================================" && \
-find $STAGING_DIR/deployment_tools/inference_engine/lib/$ARCHDIR/ -maxdepth 1 -type f -name "*.so" -exec chrpath --delete {} \; && \
-find $STAGING_DIR/deployment_tools/inference_engine/bin/$ARCHDIR/ -maxdepth 1 -type f -exec chrpath --delete {} \; && \
-find $STAGING_DIR/python/${PYTHONVER}/openvino/inference_engine/ -maxdepth 1 -type f -name "*.so" -exec chrpath --delete {} \; || \
-fail 21 "Package structure preparation failed. Stopping"
-
```

Comment:
Do we still need `#Package creation` part that follows. It also was a part of proper arranging of build artifacts which now should be already deployed to `$STAGING_DIR`

Reply:
Fixed